pkg/client/monitoring/v1/types.go

```diff
@@ -215,6 +221,45 @@ type StorageSpec struct {
 	VolumeClaimTemplate v1.PersistentVolumeClaim `json:"volumeClaimTemplate,omitempty"`
 }
 
+// ThanosSpec defines parameters for a Prometheus server within a Thanos deployment.
+// +k8s:openapi-gen=true
+type ThanosSpec struct {
+	// Peers is a DNS name for Thanos to discover peers through.
+	Peers string `json:"peers"`
+	// Version describes the version of Thanos to use.
+	Version string `json:"version"`
+	// Thanos base image if other than default.
+	BaseImage *string `json:"baseImage,omitempty"`
+	// GCS configures use of GCS in Thanos.
```

Comment:
`s/GCS configures use of GCS in Thanos./GCS configures use of Google Cloud Storage in Thanos.

Reply:
I'll make sure to expand acronyms at least once for each of them.